src/components/policy/policy_external/test/policy_manager_impl_ptu_test.cc

```diff
@@ -926,7 +926,7 @@ TEST_F(
 uint32_t GetCurrentDaysCount() {
   date_time::TimeDuration current_time = date_time::getCurrentTime();
   const uint32_t kSecondsInDay = 60 * 60 * 24;
-  return current_time.tv_sec / kSecondsInDay;
+  return date_time::getSecs(current_time) / kSecondsInDay;
```

Comment:
@LitvinenkoIra May be is better to move `date_time::getSecs(current_time)` in separate const variable?

Reply:
@ZhdanovP Could you give some arguments for this?

Reply:
@KVGrygoriev I see it useful during debug

Reply:
@KVGrygoriev Exactly, ok, closed =)